backend/src/apiserver/server/visualization_server.go

```diff
@@ -11,8 +11,15 @@ import (
 
 	"github.com/golang/glog"
 	"github.com/kubeflow/pipelines/backend/api/go_client"
+	"github.com/kubeflow/pipelines/backend/src/apiserver/common"
 	"github.com/kubeflow/pipelines/backend/src/apiserver/resource"
 	"github.com/kubeflow/pipelines/backend/src/common/util"
+	"github.com/pkg/errors"
+)
+
+const (
+	visualizationServiceName = "VisualizationServiceName"
```

Comment:
Discussion: when we use env to override this, I have to write `VISUALIZATIONSERVICENAME` (https://github.com/spf13/viper#working-with-environment-variables) Shall we change the naming to be more env var friendly like `VisualizationService.Name` and `VisualizationService.Port`? so that when using env vars, we need to write `VISUALIZATIONSERVICE_NAME` and `VISUALIZATIONSERVICE_PORT`?

Note, as we discovered before, it's better not to let env vars end with `_SERVICE_PORT`, because they may be conflicted with other Kubernetes services

Reply:
Done.